src/Media.Plugin/UWP/NewMediaImplementation.cs

```diff
@@ -481,6 +483,10 @@ Task<bool> ResizeAsync(StorageFile file, StoreCameraMediaOptions mediaOptions)
                         var finalWidth = Convert.ToUInt32(bitmap.PixelWidth * percent);
                         var finalHeight = Convert.ToUInt32(bitmap.PixelHeight * percent);
 
+                        var folder = await file.GetParentAsync();
```

Comment:
this alone solves the issue?
Deleting the file and creating a new one? :-)

Reply:
jep... 🙈